test/tools/libtesteth/TestSuite.cpp

```diff
@@ -85,11 +85,23 @@ namespace dev
 namespace test
 {
 
+string const c_fillerPostf = "Filler";
```

Comment:
You have to decide if you use `std::` or not.

Reply:
what is our code style?  
I thought .h files use std::  
while .cpp files could be with  using namespace std;

Reply:
I prefer using `std::`, but you don't have to. The problem is you are using `string` and `std::vector` in the same PR.